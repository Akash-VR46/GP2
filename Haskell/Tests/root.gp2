Main = root ; swap

root () [
	(n1, empty # blue)
	(n2, empty # red)
|
] => [
	(n1 (R), empty # blue)
	(n2, empty # red)
|
]
interface = {n1, n2}

swap () [
	(n1 (R), empty # blue)
	(n2, empty # red)
|
] => [
	(n1, empty # blue)
	(n2 (R), empty # red)
|
]
interface = {n1, n2}

