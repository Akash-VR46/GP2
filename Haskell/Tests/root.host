[
	(n1, empty # blue)
	(n2, empty # red)
|
]
