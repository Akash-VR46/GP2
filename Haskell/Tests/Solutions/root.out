[
 (n1 (R), empty # red)
 (n2, empty # blue)
|
]
